src/features/Lock/stats.js

```diff
@@ -137,19 +139,25 @@ const Stats = ({ wallet, lockedAlready }) => {
       const { tvl, lockedByUser, hiIQSupply, rewardsAcrossLockPeriod } =
         await getStats(wallet);
 
+      const aprDefault = await defaultAPR(); // TODO: move to right place (user is not logged). example to get APR without wallet connected: use value for base APR (rename to just APR)
+      console.log(aprDefault);
+
       const yearsLock = 4; // assuming a 4 year lock
 
-      const rewardsBasedOnLockPeriod = lockedByUser * (1 + 0.75 * yearsLock);
+      const amountLocked = lockedByUser > 0 ? lockedByUser : 100000;
```

Comment:
this allows to show APR to users that have no hiiq

Reply:
I would suggest to try to refactor all the APR part into his own scoped function because its adding too much complexity to the Stats function (and it will increase with the APR for unlogged users)